miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -322,6 +322,7 @@ HotTarget.sample = (function() {
               };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)